ospfd/ospf_main.c

```diff
@@ -48,14 +48,20 @@
 #include "ospfd/ospf_apiserver.h"
 
 #define OSPFD_STATE_NAME	 "%s/ospfd.json", frr_libstatedir
-#define OSPFD_INST_STATE_NAME(i) "%s/ospfd-%d.json", frr_runstatedir, i
+#define OSPFD_INST_STATE_NAME(i) "%s/ospfd-%d.json", frr_libstatedir, i
 
 /* this one includes the path... because the instance number was in the path
  * before :( ... which totally didn't have a mkdir anywhere.
+ *
```

Comment:
I don't see problems with the C logic but I'd rather see a comment explaining the usage and rationale behind the various paths. Your comment as written seems only useful in the context of what you are changing as opposed to what paths are created and what they are to be used for.

Reply:
done (but I've put it in `lib/libfrr.h` because the variables are declared there & used beyond ospfd)